slatedb/src/compactor.rs

```diff
@@ -511,13 +511,15 @@ pub mod stats {
     }
 
     pub const BYTES_COMPACTED: &str = compactor_stat_name!("bytes_compacted");
+    pub const BYTES_STORED: &str = compactor_stat_name!("bytes_stored");
```

Comment:
How about having `bytes_compacted` and `bytes_compacted_compressed` / `bytes_compacted` + `bytes_compacted_uncompressed`? It feels bit more intuitive.

Reply:
These metrics don’t have dimensions, it’s a bare bone counter. I’ve removed this new metric to avoid scope creep. We can talk about metrics some other time.